test/vamp-client/tst_PluginStub.cpp

```diff
@@ -0,0 +1,153 @@
+#include "catch/catch.hpp"
+#include "vamp-client/Loader.h"
+#include "vamp-client/PluginClient.h"
+#include "vamp-client/PluginStub.h"
+#include "vamp-support/RequestResponse.h"
+#include <vector>
+
+using namespace piper_vamp;
+using namespace piper_vamp::client;
+using AudioBuffer = std::vector<std::vector<float>>;
```

Comment:
We can see you are a truly native Modern C++ developer, because you write "using" instead of "typedef"